notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -1,1140 +1,1273 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    endpoint = model.deploy(
</pre><p>Each param should be on its own line. Ex</p><p> </p><p>result = aiplatform.method(</p><p>    param1=...,</p><p>    paramN=...</p><p>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>